crates/primitives/src/withdrawal.rs

```diff
@@ -0,0 +1,17 @@
+use crate::Address;
+use reth_codecs::{main_codec, Compact};
+use reth_rlp::{RlpDecodable, RlpEncodable};
+
+/// Withdrawal represents a validator withdrawal from the consensus layer.
+#[main_codec]
+#[derive(Debug, Clone, PartialEq, Eq, Default, RlpEncodable, RlpDecodable)]
+pub struct Withdrawal {
```

Comment:
This type also exists in 2 other places
1. https://github.com/paradigmxyz/reth/blob/e689ba03c423a32f55ace0d3a7ac5f992c60a1c7/crates/rpc/rpc-types/src/eth/engine.rs#L72
2. https://github.com/paradigmxyz/reth/blob/e689ba03c423a32f55ace0d3a7ac5f992c60a1c7/bin/reth/src/test_eth_chain/models.rs#L134

let's dedup